docs/source/host/maintain-domain/view-logs.rst

```diff
@@ -1,15 +1,10 @@
-.. warning::
-    This document is outdated.
-    FIXME: mentions High Fidelity, paths outdated, missing path for Linux servers
-
 #######################
 View Domain Server Logs
 #######################
 
 Your domain server logs list the different events relevant to your High Fidelity domain. These can help you troubleshoot any issues with your domain and understand how the different parts of your domain communicate with each other.
 
-Server logs are stored directly on the server in the following directories:
-
-* **Windows**: %AppData%/Local/High Fidelity/Server Console/logs
-* **Mac**: ~/Library/Application Support/High Fidelity/Server Console/Logs
+Server logs are stored directly on the server and can be accessed as follows:
 
+* **Windows**: Logs can be found in the directory "%AppData%\Local\Vircadia\Server Console\logs"
+* **Linux**: Logs can be accessed by running the command "journalctl -u vircadia-domain-server@default.service" for the domain server, or "journalctl -u vircadia-assignment-client@default.service" for the assignment client.
```

Comment:
```suggestion
* **Linux**: Logs can be accessed by running the command `journalctl -u vircadia-domain-server@default.service` for the domain server, or `journalctl -u vircadia-assignment-client@default.service` for the assignment client.
```

Reply:
I don't think you know what you are doing here. ` writes stuff cursive. Double `` puts stuff it in a box.

Reply:
I had originally done this with :code:`insert something here` and it was giving me odd results. Trying again it appears to work.

Reply:
should i close this one and commit something similar, but with \`these\` instead of \`\`these?\`\`

Reply:
``insert code here`` worked for me when testing so go ahead and try that locally. If it works, then it might be easiest to do it that way.

Reply:
ok, will commit then un-commit if it doesn't work

Reply:
it looks good in github, but i'm not sure how it looks after sphinx exports it.

Reply:
oh wait i read @JulianGro 's thing backwards.  never mind.